### PR TITLE
Fix radmin/eap tests

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

**Comment by jpereira** [comment]
The current code is not executing the targets `test.{eap,digest,radmin}`. But, It is working properly as can be seen below.

```
[jpereira@sugarloaf freeradius-server.git]$ make test.digest
DIGEST-TEST auth-MD5_1
DIGEST-TEST auth-MD5_2
DIGEST-TEST auth-MD5_Sess_1
DIGEST-TEST auth-MD5_Sess_2
DIGEST-TEST auth-int_1
DIGEST-TEST auth-noalgo_1
DIGEST-TEST auth_int-MD5_1
DIGEST-TEST auth_int-MD5_2
DIGEST-TEST auth_int-MD5_Sess_1
DIGEST-TEST auth_int-MD5_Sess_2
DIGEST-TEST auth_int-noalgo_1
DIGEST-TEST auth_int-noalgo_2
DIGEST-TEST md5-sess_1
DIGEST-TEST mschapv1_1
DIGEST-TEST test.example.com_1
[jpereira@sugarloaf freeradius-server.git]$
```

```
[jpereira@sugarloaf freeradius-server.git]$ make test.radmin
RADMIN-TEST help-all.txt
RADMIN-TEST help.txt
RADMIN-TEST set-module-handled-updated.txt
RADMIN-TEST show-client-all-proto.txt
RADMIN-TEST show-client-tcp-proto.txt
RADMIN-TEST show-client-udp-proto.txt
RADMIN-TEST show-config-item1.txt
RADMIN-TEST show-module-handled-status.txt
[jpereira@sugarloaf freeradius-server.git]$
```

```
[jpereira@sugarloaf freeradius-server.git]$ make test.eap
EAPOL_TEST aka
EAPOL_TEST gtc
EAPOL_TEST md5
EAPOL_TEST mschapv2
EAPOL_TEST pwd
EAPOL_TEST sim
EAPOL_TEST tls
[jpereira@sugarloaf freeradius-server.git]$
```

**Comment by jpereira** [comment]
I've fixed another issue related to https://travis-ci.org/FreeRADIUS/freeradius-server/jobs/612561378#L5213 here https://github.com/FreeRADIUS/freeradius-server/pull/3125/files#diff-36251a60226775a9b472643d2c4287bfR31

File: src/tests/radmin/config/control-socket.conf [hunk 1]
```diff
@@ -7,9 +7,9 @@
 #  Minimal radiusd.conf for testing radmin
 #
 raddb = raddb
-testdir = build/tests/radmin
-pidfile = ${testdir}/radiusd.pid
-panic_action = "gdb -batch -x src/tests/panic.gdb %e %p > ${testdir}/gdb.log 2>&1; cat ${testdir}/gdb.log"
+builddir = build/tests/radmin
```
Comment (alandekok) [inline src/tests/radmin/config/control-socket.conf]: why rename this?

Comment (jpereira) [reply src/tests/radmin/config/control-socket.conf]: Just to keep the same name used in the others conf files.

**Pull request merged**